support/doc/dependencies.md

```diff
@@ -84,6 +84,74 @@ $ sudo service redis start
 $ sudo service postgresql start
 ```
 
+## Fedora
+
+0. Upgrade your packages:
+```
+dnf upgrade
+```
+1. Add a user with sudoers group access:
+```
+useradd my-peertube-user
+passwd my-peertube-user
+usermod my-peertube-user -a -G wheel	# Add my-peertube-user to sudoers
+su my-peertube-user
+```
+2. (Optional) Install certbot (choose instructions for nginx and your distribution) :
+[https://certbot.eff.org/all-instructions](https://certbot.eff.org/all-instructions)
+3. Install NodeJS 8.x (current LTS):
+[https://nodejs.org/en/download/package-manager/#enterprise-linux-and-fedora](https://nodejs.org/en/download/package-manager/#enterprise-linux-and-fedora)
+4. Install yarn:
+[https://yarnpkg.com/en/docs/install](https://yarnpkg.com/en/docs/install)
+5. Enable [RPM Fusion](https://rpmfusion.org) for Fedora (available for x86, x86_64, armhfp)
+```
+sudo dnf install https://download1.rpmfusion.org/free/fedora/rpmfusion-free-release-$(rpm -E %fedora).noarch.rpm
+```
+This is necessary because `ffmpeg` is not in the Fedora repos.
+
+6. Run:
+```
+sudo dnf install nginx ffmpeg postgresql-server postgresql-contrib openssl gcc-c++ make redis git
+ffmpeg -version # Should be >= 3.x
+g++ -v # Should be >= 5.x
+```
+7. Post-installation
+
+_from [PostgreSQL documentation](https://www.postgresql.org/download/linux/redhat/):_
+> Due to policies for Red Hat family distributions, the PostgreSQL installation will not be enabled for automatic start or have the database initialized automatically.
+```
+# PostgreSQL
+sudo postgresql-setup initdb
+sudo systemctl enable postgresql.service
+sudo systemctl start postgresql.service
+# Nginx
+sudo systemctl enable nginx.service
```

Comment:
does nginx really need a started service on fedora?
on other distros it seems not needed

Reply:
It is not started by default, I guess it is some security policy

Reply:
According to [fedora nginx guide](https://fedoraproject.org/wiki/Nginx) you should start it after installing

Reply:
okay, good :+1: